docs/source/en/training/dreambooth.mdx

```diff
@@ -527,16 +567,57 @@ accelerate launch train_dreambooth_lora.py \
   --validation_prompt="a sks dog" \
   --validation_epochs=25 \
   --checkpointing_steps=100 \
-  --pre_compute_text_embeddings \ # Pre compute text embeddings to that T5 doesn't have to be kept in memory
-  --tokenizer_max_length=77 \ # IF expects an override of the max token length
-  --text_encoder_use_attention_mask # IF expects attention mask for text embeddings
+  --pre_compute_text_embeddings \
+  --tokenizer_max_length=77 \
+  --text_encoder_use_attention_mask
 ```
 
-### Full Dreambooth
-Due to the size of the optimizer states, we recommend training the full XL IF model with 8bit adam. 
-Using 8bit adam and the rest of the following config, the model can be trained in ~48 GB VRAM.
+### IF stage II LoRA Dreambooth
 
-For full dreambooth, IF requires very low learning rates. With higher learning rates model quality will degrade.
+`--validation_images`: These images are upscaled during validation steps.
+
+`--class_labels_conditioning=timesteps`: Pass additional conditioning to the UNet needed for stage II.
+
+`--learning_rate=1e-6`: Lower learning rate than stage I.
+
+`--resolution=256`: The upscaler expects higher resolution inputs
+
+```sh
+export MODEL_NAME="DeepFloyd/IF-II-L-v1.0"
+export INSTANCE_DIR="dog"
+export OUTPUT_DIR="dreambooth_dog_upscale"
+export VALIDATION_IMAGES="image_1.png image_2.png image_3.png image_4.png"
```

Comment:
```suggestion
export INSTANCE_DIR="dog"
export OUTPUT_DIR="dreambooth_dog_upscale"
export VALIDATION_IMAGES="image_1.png image_2.png image_3.png image_4.png"
```

what is image_1.png ... ? How can we download those . It would be nice if someone that reads the ## IF section can just execute commands to get it running

Reply:
https://github.com/huggingface/diffusers/pull/3628/commits/826d0c38c05a0720faee5ad6f5b2d34e708399c0 done here!